docs/modules/ROOT/pages/meta-updater-usage.adoc

```diff
@@ -0,0 +1,87 @@
+= Usage
+:meta-updater-github-url: https://github.com/advancedtelematic/meta-updater/tree/master
+:metadata-expiry-article: xref:dev@ota-client::metadata-expiry.adoc[OTA Connect documentation]
+ifdef::env-github[:metadata-expiry-article: link:https://docs.ota.here.com/ota-client/dev/metadata-expiry.html[OTA Connect documentation]]
+
+//MC: No overlap with any content currently in the developer guide, but probably useful content to clean up and include. eg: use cases.
+
+== OSTree
```

Comment:
FYI I'm not sure anyone has used or reviewed this OSTree section in years. It may or may not still be relevant and useful.

Reply:
Yes, I'll do a quality review and rewrite of this content in a second PR. This PR just to get it into the format/structure that Antora expects.